RELEASE/scripts/sl_ascend/sl_koe.ash

```diff
@@ -9,6 +9,7 @@ boolean koe_initializeSettings()
 {
 	if(in_koe())
 	{
+		set_property("sl_getStarKey", false);

```

Comment:
Is this property just named incorrectly? This looks like it's saying to NOT get the star key, which isn't what the commit message says.

Reply:
🤦‍♂